src/_posts/databases/postgresql/2000-01-01-troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshooting Scalingo for PostgreSQL
+nav: Troubleshooting
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 9
+---
+
+## Identifying Performances Issues
+
+
+## Understanding Idle Queries
+
+PostgreSQL® tends to use all the available memory if there is enough indices to
+fill the memory. If there is too many indices to fit into the memory, some of
+them are stored on the disk. In this situation, queries needing these indices
+will be slowed down. PostgreSQL® first needs to load the indices from the disk
+into the RAM which takes some time. The memory usage on the "Metrics" tab of
+your web dashboard would always be at 100% in such situation.
```

Comment:
suggestion: maybe emphasize that this is not necessarily a problem that 100% of the RAM is used. If indices stored on the disk aren't often used, it's perfectly fine to have only a few slower queries.